deepchecks/checks.py

```diff
@@ -34,7 +34,6 @@
     'SpecialCharacters',
     'StringLengthOutOfBounds',
```

Comment:
Isnt all of this file to be deprecated?

Reply:
yes but not now. We decided to keep the ability to import tabular stuff directly from deepcheck until 0.10.0